metadata.rb

```diff
@@ -5,13 +5,13 @@
 license          'Apache 2.0'
 description      'Installs/Configures elkstack'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '4.2.5'
+version          '4.2.6'
 
 depends 'apt'
 depends 'build-essential'
 depends 'chef-sugar'
 depends 'cron', '~> 1.4.3'
-depends 'elasticsearch', '~> 0.3'
+depends 'elasticsearch', '>= 0.3.7'
```

Comment:
I don't love modifying the pin to an aggressive minimum. Should be able to leave it as is so long as the version used is > 0.3 < 0.4